bigchaindb/commands/bigchaindb.py

```diff
@@ -10,22 +10,17 @@
 import sys
 
 from bigchaindb.common import crypto
-from bigchaindb.common.exceptions import (StartupError,
-                                          DatabaseAlreadyExists,
+from bigchaindb.common.exceptions import (DatabaseAlreadyExists,
                                           KeypairNotFoundException,
                                           DatabaseDoesNotExist)
 import bigchaindb
 from bigchaindb.tendermint.core import BigchainDB
 from bigchaindb import backend
 from bigchaindb.backend import schema
 from bigchaindb.backend import query
-from bigchaindb.backend.admin import (set_replicas, set_shards, add_replicas,
-                                      remove_replicas)
-from bigchaindb.backend.exceptions import OperationError
 from bigchaindb.commands import utils
 from bigchaindb.commands.messages import (
     CANNOT_START_KEYPAIR_NOT_FOUND,
-    RETHINKDB_STARTUP_ERROR,
```

Comment:
I think _maybe_ you can remove the import of `CANNOT_START_KEYPAIR_NOT_FOUND` as well, including the call (below) that uses it. A BigchainDB instance doesn't have any internal keypair any more, so `KeypairNotFoundException` is probably defunct and can also be deleted everywhere in all the code. Maybe such keypair-related changes could be done in a separate pull request, since it's _a different concern_ that's only peripherally related to the CLI changes (or is it?). There could be a PR to scrub all code related to the BigchainDB node's keypair. A simpler principle to apply might just be: "make lots of small pull requests, each with narrow scope."

export_my_pubkey is definitely _in_ scope since that's a CLI command.

Reply:
Makes sense, I think addressing this in a separate PR is a better option.